authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,615 @@
+
+.STS authentication flow
+image::sts_flow.png[STS authentication flow]
+
+The implementation of STS mode changes the format of the AWS credentials root secret in the `kube-system` namespace.
+
+.AWS secret format using a mode other than STS
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: aws-creds
+data:
+  aws_access_key_id: <base64-encoded-access-key-id>
+  aws_secret_access_key: <base64-encoded-secret-access-key>
+----
+
+.AWS secret format using STS mode
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: aws-creds
+data:
+  role_name: <operator-role-name> <1>
+  web_identity_token_file: <path-to-token> <2>
+----
+<1> The IAM role for the cluster
++
+<needinfo> Is this accurate/informative? or is there a better way to describe it?
+<2> The path to the service account tokens.
+
+[id="sts-mode-installing"]
+== Installing an {product-title} cluster configured for STS mode
+
+In {product-title} version 4.7, installing a cluster that is configured to use the CCO in STS mode requires the following steps:
+
+. xref:#sts-mode-installing-preparing[Preparing the installer]
+. Creating AWS resources xref:#sts-mode-installing-config-preflight[using the `sts-preflight` tool] or xref:#sts-mode-installing-config-manual[manually]
+. xref:#sts-mode-installing-running-installer[Running the installer]
+. xref:#sts-mode-installing-verifying[Verifying the installation]
+
+[NOTE]
+====
+<needinfo>
+Are they basically following the link:https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-aws-customizations.html#installation-initializing_installing-aws-customizations[Installing a cluster on AWS with customizations] procedure here, with some specific differences for STS (and of course the addition of the preflight stuff)?
+====
+
+
+[id="sts-mode-installing-preparing"]
+=== Preparing the installer
+
+.Prerequisites
+* Obtain the pull secret for your cluster.
+
+.Procedure
+
+. Set the `$RELEASE_IMAGE` environment variable to use a sufficiently new {product-title} release:
++
+[source,terminal]
+----
+<needinfo>
+- Need information about how to do this (commands, output if applicable)
+- Is it this step? https://docs.openshift.com/container-platform/4.6/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#retrieving-the-openshift-installer_ipi-install-installation-workflow
+- Will also need to know more specifically about "sufficiently new".
+----
+
+. Extract the AWS `CredentialsRequest` objects from the release image:
++
+[source,terminal]
+----
+$ mkdir credreqs ; oc adm release extract --cloud=aws --credentials-requests $RELEASE_IMAGE --to=./credreqs ; cat ./credreqs/*.yaml > credreqs.yaml
+----
+
+. Extract the {product-title} installer from the release image:
++
+[source,terminal]
+----
+$ oc adm release extract --command=openshift-install $RELEASE_IMAGE
+----
```

Comment:
Again, this is a step from back before all the PRs had merged. This shouldn't be necessary, as the user can just download the installer from whatever regular location they normally get it from.